frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tsx

```diff
@@ -86,11 +124,15 @@ const Advanced = ({
           appConfig.smtp_settings?.authentication_method || "",
         domain,
         verify_ssl_certs: verifySSLCerts,
-        enable_start_tls: enableStartTLS,
+        enable_start_tls: enableStartTLS || false,
       },
       host_expiry_settings: {
         host_expiry_enabled: enableHostExpiry,
-        host_expiry_window: Number(hostExpiryWindow),
+        host_expiry_window: hostExpiryWindow || undefined,
```

Comment:
Since `hostExpiryWindow` is always a number this `||` just converts the `0` case to `undefined` right?